### PR TITLE
Emis yard docs

### PR DESCRIPTION
**Pull request by lihanli**
## Description of change

emis yard docs
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17801

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: lib/emis/responses/get_deployment_response.rb [hunk 1]
```diff
@@ -5,12 +5,16 @@
...
+    # EMIS get deployments response
     class GetDeploymentResponse < EMIS::Responses::Response
+      # (see EMIS::Responses::GetCombatPayResponse#item_tag_name)
       def item_tag_name
         'deployment'
       end
 
       # rubocop:disable Metrics/MethodLength
+
+      # (see EMIS::Responses::GetCombatPayResponse#item_schema)
```
Comment (hpjaj) [inline lib/emis/responses/get_deployment_response.rb]: Perhaps use the [`@see`](https://www.rubydoc.info/gems/yard/file/docs/Tags.md#see) tag for all of these?

Comment (lihanli) [reply lib/emis/responses/get_deployment_response.rb]: using `(see)` copies the documentation from the other method, seems easier than linking to it

File: lib/emis/models/deployment_location.rb [hunk 2]
```diff
@@ -13,6 +14,8 @@ class DeploymentLocation
       attribute :termination_reason_code, String
```
Comment (jholton) [inline lib/emis/models/deployment_location.rb]: Do we want to document these Virtus attributes? I have been, copying the way @kreek did it [here](https://github.com/department-of-veterans-affairs/vets-api/pull/2899/files).

Comment (lihanli) [reply lib/emis/models/deployment_location.rb]: i'll add it in another pr

**Pull request merged**